### DISCUSSION TITLE
About #1925

### DISCUSSION THREAD

**Discussion by ghost** [pyscript/pyscript#1928] [Q&A]
What's the difference between preview page and site page?preview page doesn't display result where as site page does for worker.
why is it?
@JeffersGlass @WebReflection @madhur-tandon 
<img width="446" alt="1" src="https://github.com/pyscript/pyscript/assets/79505893/38f54c44-34cf-450a-99d9-7f5770dbd0d4">
<img width="379" alt="2" src="https://github.com/pyscript/pyscript/assets/79505893/eaf2f121-85e3-4aba-ad83-67a9c9d36ecb">

**Comment by WebReflection** [comment]
IIRC we're working to fix preview pages and workers because there are difference sources and constraints due SharedArrayBuffer requirements. The live page doesn't have such issue because no iframe is used at all. Some browser might also work regardless, some other might have extra constraints but, like I've said, I think we're working on it to finally resolve that discrepancy among the two pages.
